include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //uint32_t           advancedBlendMaxColorAttachments;
+    //VkBool32           advancedBlendIndependentBlend;
+    //VkBool32           advancedBlendNonPremultipliedSrcColor;
+    //VkBool32           advancedBlendNonPremultipliedDstColor;
+    //VkBool32           advancedBlendCorrelatedOverlap;
+    //VkBool32           advancedBlendAllOperations;
+            
+    /* ConservativeRasterizationPropertiesEXT *//* provided by VK_EXT_conservative_rasterization */
+    //float              primitiveOverestimationSize;
+    //float              maxExtraPrimitiveOverestimationSize;
+    //float              extraPrimitiveOverestimationSizeGranularity;
+    //VkBool32           primitiveUnderestimation;
+    //VkBool32           conservativePointAndLineRasterization;
+    //VkBool32           degenerateTrianglesRasterized;
+    //VkBool32           degenerateLinesRasterized;
+    //VkBool32           fullyCoveredFragmentShaderInputVariable;
+    //VkBool32           conservativeRasterizationPostDepthCoverage;
+            
+    /* CustomBorderColorPropertiesEXT *//* provided by VK_EXT_custom_border_color */
+    //uint32_t           maxCustomBorderColorSamplers;
+
+    /* DescriptorIndexingPropertiesEXT ---> MOVED TO Vulkan 1.2 Core  *//* provided by VK_AMD_shader_core_properties */
+
+    /* DiscardRectanglePropertiesEXT *//* provided by VK_EXT_discard_rectangles */
+    //uint32_t           maxDiscardRectangles;
+            
+    /* ExternalMemoryHostPropertiesEXT *//* provided by VK_EXT_external_memory_host */
+    //VkDeviceSize       minImportedHostPointerAlignment;
+    
+    /* FragmentDensityMapPropertiesEXT *//* provided by VK_EXT_fragment_density_map */
+    //VkExtent2D         minFragmentDensityTexelSize;
+    //VkExtent2D         maxFragmentDensityTexelSize;
+    //VkBool32           fragmentDensityInvocations;
+    
+    /* FragmentDensityMap2PropertiesEXT *//* provided by VK_EXT_fragment_density_map2 */
+    //VkBool32           subsampledLoads;
+    //VkBool32           subsampledCoarseReconstructionEarlyAccess;
+    //uint32_t           maxSubsampledArrayLayers;
+    //uint32_t           maxDescriptorSetSubsampledSamplers;
+    
+    /* GraphicsPipelineLibraryPropertiesEXT *//* provided by VK_EXT_graphics_pipeline_library */
+    //VkBool32           graphicsPipelineLibraryFastLinking;
+    //VkBool32           graphicsPipelineLibraryIndependentInterpolationDecoration;
+
+    /* InlineUniformBlockPropertiesEXT ---> MOVED TO Vulkan 1.3 Core  */
+
+    /* LineRasterizationPropertiesEXT *//* provided by VK_EXT_line_rasterization */
+    //uint32_t           lineSubPixelPrecisionBits;
+
+    /* MultiDrawPropertiesEXT *//* provided by VK_EXT_multi_draw */
+    //uint32_t           maxMultiDrawCount;
+
+    /* PCIBusInfoPropertiesEXT *//* provided by VK_EXT_pci_bus_info */
+    //uint32_t           pciDomain;
+    //uint32_t           pciBus;
+    //uint32_t           pciDevice;
+    //uint32_t           pciFunction;
```

Comment:
find out what value is invalid (what value cannot be taken by at least one of these fields) to find a good default

treat GL/GLES like a Vulkan implementation which doesn't support the extension

Reply:
mine reports 0, 9, 0, 0 respectively, I guess setting pciDevice to `~0u` as default is a good indicator for now